### DISCUSSION TITLE
Generation mask of MAISI mask generation autoencoderKL

### DISCUSSION THREAD

**Discussion by Wlllllllly** [Project-MONAI/MONAI#8174] [General]
I am very curious about the mask generation process in the MAISI mask generation AutoencoderKL model. I noticed that the in_channel is set to 8 and the out_channel is 125. My understanding is that the output likely corresponds to masks for 125 labels. However, I am unsure about the exact input format. Could you please explain the training method used for the mask generation? I would greatly appreciate any insights you could provide.

**Comment by KumoLiu** [comment]
Hi @Wlllllllly, thanks for your interest here.
You can find more information here: https://github.com/Project-MONAI/tutorials/tree/main/generation/maisi#3-model-training.
If meet any further questions, feel free to ask. Thanks.
